10.yaml

```
@@ -0,0 +1,49 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: q10
+  labels:
+    question.kontest.amsy.dev: "10"
+data:
+  question: |-
+    Kubernetesで永続化領域の要求を行うリソースが、PersistentVolumeClaimです。永続化領域の要求が発行されるとKube-Schedulerが保持しているPersistentVolumeから最適なVolumeを割り当てます。
+    800Mi以上のボリュームを要求し、RWXのアクセス モードを指定するcndt-pvcという名前のPersistentVolumeClaimとなるように以下マニフェストファイルを変更してください。
```

```suggestion
    800Mi以上のボリュームを要求し、RWXのアクセスモードを指定するように以下のマニフェストファイルを変更してください。
```

以下のマニフェスト、とあるので名前やリソース名までは書かなくても良さそうかなと思いました